Tensile/TensileCreateLibrary.py

```diff
@@ -190,10 +190,18 @@ def isSupported(arch):
         print('hipcc:', ' '.join(compileArgs))
       subprocess.check_call(compileArgs)
 
+      infile = os.path.join(buildPath, objectFilename)
+      try:
+        bundlerArgs = [globalParameters["ClangOffloadBundlerPath"], "-type=o", "-inputs=%s" % infile, "-list"]
+        buffer = subprocess.check_output(bundlerArgs, stderr=subprocess.STDOUT).decode()
+        # TODO - use the list of gfx entries returned as targets directly
+        hipVersion = "hipv4" if "hipv4" in buffer else "hip"
```

Comment:
I am OK with this quick workaround. I hope you can have a better fix when you get time since the bundle ID may change again in the future.